examples/official-storybook/built-storybooks/riot-kitchen-sink

```diff
@@ -0,0 +1 @@
+../../riot-kitchen-sink/storybook-static
```

Comment:
This should be a symlink not a text file

Reply:
quite hard with windows, will try with linux instead